webapp/components/PageTransition.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Flex } from "@chakra-ui/react";
+import { motion } from "framer-motion";
+
+interface PageTransitionProps {
+  children: React.ReactNode;
+}
+
+const PageTransition = ({ children }: PageTransitionProps) => (
```

Comment:
It's ok. I think that:
`const MyComponent: React.FC<{}> = ({ children } => {
return <Component />
}`
This way you make implicit the use of children in props and there is no need of add the extra interface

Reply:
If you need extra props you add them inside  `React.FC<IExtraProps>`